src/data/lovelace.ts

```diff
@@ -1,5 +1,10 @@
-import { HomeAssistant } from "../../../types";
-import { LovelaceConfig, LovelaceCardConfig } from "../types";
+import { HomeAssistant } from "../types";
+import { LovelaceConfig, LovelaceCardConfig } from "../panels/lovelace/types";
+
+export const lovelaceConfig = (hass: HomeAssistant): Promise<void> =>
```

Comment:
Should we call this `fetchConfig`? Also, returning a `Promise<void>`? That should be `Promise<LovelaceConfig>` ?